modules/fedimint-mint/src/config.rs

```diff
@@ -15,13 +15,15 @@ pub struct MintConfig {
     pub peer_tbs_pks: BTreeMap<PeerId, Tiered<tbs::PublicKeyShare>>,
     pub fee_consensus: FeeConsensus,
     pub threshold: usize,
+    pub target_denomination_sets: u16,
```

Comment:
This doesn't seem to be something the federation really cares about. I think the client config generation should have a hard coded value and you can edit the field in the client config to change it.

Reply:
Sure, I was wondering if this could act as a maximum acceptable notes as change on the federation side, but we could implement that later.

Reply:
Great idea, we should do that! But I think in that case it should be named differently.